sdcm/cluster.py

```diff
@@ -1644,9 +1644,14 @@ def decode_raw_backtrace(self, scylla_debug_file, raw_backtrace):
 
     def get_scylla_build_id(self) -> Optional[str]:
         for scylla_executable in ("/usr/bin/scylla", "/opt/scylladb/libexec/scylla", ):
-            output = self.remoter.run(f"readelf -n {scylla_executable}", ignore_status=True).stdout
-            if match := BUILD_ID_RE.search(output):
-                return match.group("build_id")
+            if self.distro.is_rhel_like:
+                output = self.remoter.run(f"readelf -n {scylla_executable}", ignore_status=True).stdout
+                if match := BUILD_ID_RE.search(output):
+                    return match.group("build_id")
+            elif self.distro.is_ubuntu:
+                build_id_result = self.remoter.run(f"{scylla_executable} --build-id", ignore_status=True)
```

Comment:
Why do we need 2 different commands to read build-id. 
On centos the same command scylla_exec --build-d doesn't return the same result as for ubuntu, does it?

Reply:
you are right... i tested on a CentOS instance, and it returns the same result...
updated the code..